### ISSUE TITLE
[Feature] Focus & Highlight current playing song in playlist

### ISSUE THREAD

**Issue by Gorge-1** [jurialmunkey/skin.arctic.horizon.2#902]
### Skin section

Music Visualisation OSD

### Feature description

While playing a playlist and pressing DOWN in the Visualisation OSD we can see the playlist.
When we press DOWN the currently playing song should focus & highlight in the playlist.

### Why is this feature needed? Please justify and explain.

Better navigation of the playlist. See the currently playing song and the previous / next songs.

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
>  Other skins do this aswell.

Which skins?

**Comment by Gorge-1** [comment]
For example in the standard Estuary and the Arctic Zephyr Reloaded skins.

**Comment by jurialmunkey** [comment]
Estuary doesn't have a playlist on the osd?

**Comment by Gorge-1** [comment]
You are right, the playlist is not directly integrated into the OSD. A button has to be pressed to go to the playlist, where the current song is focussed and highlighted.
Is it possible to create the same mechanic in the Arctic Horizon 2 - OSD playlist?

**Comment by ceefaxer** [comment]
I’d second this, mechanic. When playing and album or playlist. Having the track highlighted that is playing would be really useful.
